integrations/convex/manifest.json

```
@@ -0,0 +1,70 @@
+{
+  "name": "Convex",
+  "version": "0.0.1",
+  "overview": "Sync your LaunchDarkly feature flags to your Convex deployment.",
+  "description": "Convex is the open-source backend for application builders. An all-in-one platform with thoughtful, product-centric APIs.",
+  "author": "Convex",
+  "supportEmail": "support@convex.dev",
+  "links": {
+    "site": "https://convex.dev",
+    "privacyPolicy": "https://convex.dev/privacy"
+  },
+  "categories": ["infrastructure"],
```

```suggestion
  "categories": ["infrastructure"],
  "icons": {
      "square": "assets/images/square.svg",
      "horizontal": "assets/images/horizontal.svg"
    },
```

You'll need to specify the paths to the icons. To validate the manifest you you can run `npm run test`.

---

whoops! I thought i had those specified